src/main/java/Utils/JsonReader.java

```diff
@@ -0,0 +1,60 @@
+package Utils;
+
+import java.io.*;
+import java.net.*;
+import java.nio.charset.*;
+
+import org.apache.commons.io.*;
+import org.json.*;
+
+public class JsonReader {
+
+	private static String readAll(Reader rd) throws IOException {
+//    String s = IOUtils.toString(rd), $ = s.substring(s.indexOf('\n') + 1);
+//    return $.substring(0, $.lastIndexOf('\n'));
+		return IOUtils.toString(rd);
+	}
+
+//Had to used suppress warning since the address is always available and finally close has to happen
+	@SuppressWarnings("resource")
```

Comment:
Please resolve the warning rather then suppressing it

Reply:
done 👍